benchmark/misc/dotenv.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const { createBenchmark } = require('../common.js');
+const path = require('path');
+
+const bench = createBenchmark(main, {
+  n: 3e4,
+});
+const file = path.resolve(__dirname, '../fixtures/valid.env');
+
+function main({ n }) {
+  bench.start();
+  for (let i = 0; i < n; i++) {
+    process.loadEnvFile(file);
```

Comment:
```suggestion
#    process.loadEnvFile(file);
```
util.parseEnvFile is a better function to benchmark IMO

Reply:
How does it look now?